index.html

```diff
@@ -152,45 +152,747 @@
+          User Roles specify the roles different <a>entities</a> play when interacting
+          with <a>Verifiable Credentials</a>. Tasks define the functions users can
+          accomplish, and sequences demonstrate how tasks might be realized, by
+          interactions between <a>entities</a> over time.
+      </p>
+      <p>
+          As with all models, this use case model is neither exhaustive nor complete. The
+          listed uses cannot capture all possible use cases. Similarly, the models do not
+          completely characterize the use cases represented. However, the combined model
+          is intended to provide specific, coherent guidance for the work ahead.
+      </p>
+      <p>
+        The model used here draws heavily from the Verifiable Credentials Use Cases model.
+        This is due to the fact that the VC-API is meant to support the transport of 
+        VCs in the use cases considered for the Verfiable Credential Working Group.
       </p>
     </section>
   </section>
-
+  
   <section>
-    <h2>Use Cases</h2>
-    <p>New cases to be aggregated here, from VC and DID Use Case documents.
-    </p>
+    <h2>User Roles</h2>
+      <p>
+        There are four roles supported by <a>Verifiable Credentials</a> as defined by the 
+        VCWG: <a>Issuer</a>,
+        <a>Verifier</a>, <a>Subject</a>, and <a>Holder</a>.
+      </p>  
+      <p>
+        The above roles were those defined for the VCWG, however, due to the way most 
+        production software on the internet works, for the purposes of the VC-API there arises
+        the need to delve into some detail as to how the Issuer and Verifier roles are structured.
+        This is due to the fact that there are many use cases where one company is making use of 
+        services to handle part of the technology stack for them. This has lead to both Verifier
+        and Issuer splitting into two entities each: <a>Issuer Service</a>, <a>Issuer Coordinator</a>
+        <a>Verifier Service</a>, and <a>Verifier Coordinator</a>. 
+      </p>
+      <p>
+        This leads to the full set of roles being considered in the VC-API as: 
+        <a>Issuer Service</a>, <a>Issuer Coordinator</a>, <a>Verifier Service</a>, <a>Verifier Coordinator</a>,
+        <a>Subject</a>, and <a>Holder</a>.
+        It is fully expected that software will exist that takes on multiple roles in the
+        set of: <a>Issuer Service</a>, <a>Issuer Coordinator</a>
+        <a>Verifier Service</a>, and <a>Verifier Coordinator</a>. 
+      </p>
+      <figure>
+        <img alt="VC-API User Roles" 
+           style="display:block; margin:auto"
+           src="user-roles.png" />
+        <figcaption>
+          VC-API User Roles
+        </figcaption>
+      </figure>
+
+    <dl>
+      <dt>
+        Issuer Service
+      </dt>
+      <dd>
+        The <a>entity</a> that creates a <a>claim</a> and associates it with a
+        particular <a>subject</a>.
```

Comment:
I don't think this is correct. As I understand it, the Issuer Service (software) delivers a VC, which contains the claims made by the Issuer Coordinator (human, possibly plus software) about one or more subjects.

Reply:
Updated language to reflect this, new language is in the following commit:

772a31769b8263ff00adbb591110c2e31c7ce198